test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +249,30 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest-" + string(uuid.NewUUID())
+		_, err := f.ClientSet.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
+			},
+		})
```

Comment:
Why don't you use `f.CreateNamespace()` like the other e2e tests instead of calling the raw API?

Reply:
Updated in bd6550fb349